src/components/BooksItem.jsx

```diff
@@ -17,26 +17,55 @@ const BooksItem = ({
     dispatch(removeBook(bookKey)).then(() => dispatch(fetchBooks()));
   };
   return (
-    <div className="border p-4">
-      <span>
-        <span className="font-bold">Category:</span>
-        {' '}
-        <b>{category}</b>
-      </span>
-      <h2>
-        <span className="font-bold">Title</span>
-        {title}
-      </h2>
-      <span>
-        <span className="font-bold">Author</span>
-        {author}
-      </span>
-      <Button
-        name="Delete"
-        title="delete"
-        onClick={handleDeleteBook}
-        className="border ml-4 px-4 py-1"
-      />
+    <div className="relative max-w-[1175px] m-auto my-4 py-7 px-1.5 border bg-white flex justify-between md:p-4">
+      <div>
+        <span className="text-xs font-bold text-slate-400">{category}</span>
+        <h2 className="text-sm font-bold -mt-2 -mb-2 md:text-2xl">{title}</h2>
+        <span className="text-xs text-primary">{author}</span>
+        <div className="flex gap-2 mt-4 absolute bottom-0 left-1/2 -translate-x-1/2 border-t-2 shadow-inner md:relative md:left-0 md:translate-x-0 md:border-none md:shadow-none">
+          <Button
+            name="Comment"
+            title="Comment"
+            onClick={() => null}
+            className="text-xs text-primary px-2 border-r-0 focus:font-bold hover:font-bold md:pl-0 md:border-r-2"
+          />
+          <Button
+            name="Edit"
+            title="Edit"
+            onClick={() => null}
+            className="text-xs text-primary px-2 border-r-0 focus:font-bold hover:font-bold md:pl-0 md:border-r-2"
+          />
+          <Button
+            name="Delete"
+            title="delete"
+            onClick={handleDeleteBook}
+            className="text-xs text-primary px-2 border-r-0 focus:font-bold hover:font-bold md:pl-0 md:border-r-2"
+          />
+        </div>
+      </div>
+      <div className="flex items-center gap-4 md:mx-10">
+        <div className="relative w-12 h-12 rounded-full border-4 border-primary flex justify-center items-center">
+          <span className="font-roboto md:absolute left-12">
+            <span className="text-xs md:text-3xl">0%</span>
+            <br />
+            <span className="text-xs text-slate-400 hidden md:block">
+              Completed
+            </span>
+          </span>
+        </div>
```

Comment:
- [ ] Kindly ensure leaving enough gap between the `Progressive Circle` and the progress information in order to match the provided Zeplin design.

   - ### YOUR DESIGN
 
      ![chrome-capture-2023-8-30 (2)](https://github.com/lily4178993/bookstore-cms/assets/87197412/020e3933-5b31-4083-931a-dc95f21992fa)

   - ### PROVIDED ZEPLIN DESIGN

  
    ![chrome-capture-2023-8-30 (4)](https://github.com/lily4178993/bookstore-cms/assets/87197412/30a244a8-0f74-4045-b6d6-78f13253a94b)